### PR TITLE
AF-2625: Create Liveness and Readiness Probe on Dashbuilder Runtimes

### PR DESCRIPTION
**Pull request by jesuino**
**Thank you for submitting this pull request**

**JIRA**:
[AF-2625](https://issues.redhat.com/browse/AF-2625)

I think that initially this is all for the ready and liveness probes.

*/rest/ready*: Indicates that the server is ready to be accessed;
*/rest/healthy*: Successful response only if the server is ready.

### CODE REVIEW

**Comment by jesuino** [comment]
jenkins execute fdb

**Review by caponetto — APPROVED** [review]

**Comment by jesuino** [comment]
@spolti 

Could you please let us know if this enough for liveness and health check for dashbuilder runtime images?

THanks!

**Comment by tomasdavidorg** [comment]
jenkins do fdb

File: dashbuilder/dashbuilder-runtime/src/main/java/org/dashbuilder/backend/health/HealthService.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+/**
+ * Service that reports if Runtime is ready and healthy.
+ *
+ */
+@Path("/")
+public class HealthService {
+    
+    private static String SUCCESS_RESPONSE = "{\"success\": true}";
```
Comment (tomasdavidorg) [inline dashbuilder/dashbuilder-runtime/src/main/java/org/dashbuilder/backend/health/HealthService.java:33]:
```suggestion
    private static final String SUCCESS_RESPONSE = "{\"success\": true}";
```

**Comment by tomasdavidorg** [comment]
Are those failed tests related or random failures?

**Review by tomasdavidorg — APPROVED** [review]
Thanks @jesuino . Looks good to me :slightly_smiling_face:

**Review by ederign — APPROVED** [review]

**Pull request merged**